Header header
uint32 joint			#which joint the graph sets
motoman_sia20d_moveit_updated/trajPlotPoint[] points
